From 3f6f1fb35c2aa14a40d70b4673bae75fc9fee1e2 Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <l.majewski@samsung.com>
Date: Tue, 8 Oct 2013 11:50:35 +0200
Subject: [PATCH 0843/1302] regulator:dts:slp-PD: MAX8997 device tree nodes
 definitions for MIDAS (PEGASUS_D)

Adjustment of regulator configuration (max8997) for MIDAS (PEGASUSD) device.

Signed-off-by: Lukasz Majewski <l.majewski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4212-slp_pd.dts | 50 ++++++++++++++++-----------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/arch/arm/boot/dts/exynos4212-slp_pd.dts b/arch/arm/boot/dts/exynos4212-slp_pd.dts
index 4b2e573..7d658f3 100644
--- a/arch/arm/boot/dts/exynos4212-slp_pd.dts
+++ b/arch/arm/boot/dts/exynos4212-slp_pd.dts
@@ -143,23 +143,23 @@
 			max8997,pmic-buck125-default-dvs-idx = <0>;
 
 			max8997,pmic-buck125-dvs-gpios = <&gpj1 1 0>,
-						         <&gpj1 2 0>,
+							 <&gpj1 2 0>,
 							 <&gpl0 0 0>;
 
-			max8997,pmic-buck1-dvs-voltage = <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>;
+			max8997,pmic-buck1-dvs-voltage = <1350000>, <1300000>,
+							 <1250000>, <1200000>,
+							 <1150000>, <1100000>,
+							 <1000000>, <950000>;
 
-			max8997,pmic-buck2-dvs-voltage = <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>;
+			max8997,pmic-buck2-dvs-voltage = <1100000>, <1000000>,
+							 <950000>,  <900000>,
+							 <1100000>, <1000000>,
+							 <950000>,  <900000>;
 
-			max8997,pmic-buck5-dvs-voltage = <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>,
-							 <1100000>, <1100000>;
+			max8997,pmic-buck5-dvs-voltage = <1200000>, <1200000>,
+							 <1200000>, <1200000>,
+							 <1200000>, <1200000>,
+							 <1200000>, <1200000>;
 
 			regulators {
 
@@ -186,8 +186,8 @@
 
 				ldo4_reg: LDO4 {
 					regulator-name = "VABB02_1.8V_AP";
-					regulator-min-microvolt = <1000000>;
-					regulator-max-microvolt = <1000000>;
+					regulator-min-microvolt = <1800000>;
+					regulator-max-microvolt = <1800000>;
 					regulator-always-on;
 				};
 
@@ -226,8 +226,8 @@
 
 				ldo10_reg: LDO10 {
 					regulator-name = "VPLL_1.0V_AP";
-					regulator-min-microvolt = <1000000>;
-					regulator-max-microvolt = <1000000>;
+					regulator-min-microvolt = <1800000>;
+					regulator-max-microvolt = <1800000>;
 					regulator-always-on;
 				};
 
@@ -288,23 +288,23 @@
 				};
 
 				buck1_reg: BUCK1 {
-					regulator-name = "VARM_1.0V_AP";
-					regulator-min-microvolt = <1000000>;
-					regulator-max-microvolt = <1000000>;
+					regulator-name = "vdd_arm";
+					regulator-min-microvolt = <900000>;
+					regulator-max-microvolt = <1350000>;
 					regulator-always-on;
 				};
 
 				buck2_reg: BUCK2 {
-					regulator-name = "VINT_1.0V_AP";
-					regulator-min-microvolt = <1000000>;
-					regulator-max-microvolt = <1000000>;
+					regulator-name = "vdd_int";
+					regulator-min-microvolt = <900000>;
+					regulator-max-microvolt = <1100000>;
 					regulator-always-on;
 				};
 
 				buck3_reg: BUCK3 {
 					regulator-name = "VG3D_1.0V_AP";
-					regulator-min-microvolt = <1000000>;
-					regulator-max-microvolt = <1000000>;
+					regulator-min-microvolt = <850000>;
+					regulator-max-microvolt = <1100000>;
 					regulator-always-on;
 				};
 
-- 
1.8.3.2

